recipes/zbar-py/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "zbar-py" %}
+{% set version = "1.0.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 8f90fa36480f50109a8555a3e4debed69f70d8085545fd413190ae21a1455466
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True     # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```

Comment:
You shouldn't need 'cxx' here. The package only has a C extension, no C++ extension

Reply:
Updated as per suggestion